src/data/bucket/symbol_bucket.js

```diff
@@ -609,6 +614,7 @@ class SymbolBucket implements Bucket {
                lineOffset: [number, number],
                alongLine: boolean,
                feature: SymbolFeature,
+               layer: SymbolStyleLayer,
```

Comment:
`layer` is no longer used, so this is a leftover right?

Reply:
Oh, didn't catch that! Fixed 🙂